### PR TITLE
Fixed readme to specify paths can be relative

### PR DESCRIPTION
**Pull request by tpuhan**
I retried the relative paths and it works now. I must've been wrong back then, but I am sure now that the relative paths work. I updated the README to reflect this.

### CODE REVIEW

**Review by agrawal-siddharth — CHANGES_REQUESTED** [review]
Can you also convert the existing absolute paths found in the files expipeline.conf, plugins.conf and tanip_pipeline.conf to be relative as part of this PR? Make sure the pipelines  still work with the relative paths.

File: app/logfile.log [hunk 1]
```diff
@@ -49,3 +49,7 @@
 {"Time":"2024-06-28T21:26:17Z","Text":"Hello World!"}
 {"Time":"2024-06-28T21:26:18Z","Text":"Hello World!"}
 {"Time":"2024-06-28T21:26:19Z","Text":"Hello World!"}
+{"Time":"2024-07-09T20:36:31Z","Text":"Hello World!"}
```
Comment (agrawal-siddharth) [inline app/logfile.log]: Can changes to this file be omitted from this PR?

**Review by agrawal-siddharth — CHANGES_REQUESTED** [review]

File: app/logfile.log (hunk 1 above)
Comment (tpuhan) [reply app/logfile.log]: I'm not really sure but I can try!

Comment (tpuhan) [reply app/logfile.log]: Figured it out!

**Review by agrawal-siddharth — APPROVED** [review]

**Pull request merged**